references.bib

```diff
@@ -0,0 +1,138 @@
+@misc{Agile,
+    title={Principles behind the Agile Manifesto},
+    author={{Kent Beck, Mike Beedle, Arie van Bennekum, Alistair Cockburn, Ward Cunningham, Martin Fowler, James Grenning, Jim Highsmith, Andrew Hunt, Ron Jeffries, Jon Kern, Brian Marick, Robert C. Martin, Steve Mellor, Ken Schwaber, Jeff Sutherland, Dave Thomas}},
+    howpublished={[Online]. Available: \url{http://agilemanifesto.org/principles.html}},
+    note={[Accessed: 22-Nov-2016]}
+}
+
+@inproceedings{EnhancePerformance,
+  title={Enhancing the performance of software development virtual teams through the use of agile methods: a pilot study},
+  author={Nevo, Saggi and Chengalur-Smith, InduShobha},
+  booktitle={System Sciences (HICSS), 2011 44th Hawaii International Conference on},
+  pages={1--10},
+  year={2011},
+  organization={IEEE}
+}
```

Comment:
Month and address seem to be missing from the conference entries. Look these up and amend.